zipkin-anormdb/build.gradle

```diff
@@ -0,0 +1,16 @@
+// Database drivers
+ext.anormDriverDependencies = [
```

Comment:
Not sure I understand why this list needs to be declared here instead of just as enumerated dependencies below